pylintrc

```diff
@@ -1,7 +1,13 @@
 [MASTER]
-ignore-patterns=test_*
+ignore-patterns=test_*,conftest
 reports=no
 
+# PYLINT DIRECTORY BLACKLIST
+ignore=_generated,samples,examples,test,tests,doc,.tox
+
+init-hook='import sys; sys.path.insert(0, os.path.abspath(os.getcwd().split("azure-sdk-for-python")[0] + "azure-sdk-for-python/scripts/pylint_custom_plugin"))'
```

Comment:
`.rsplit("azure-sdk-for-python", maxsplit=1)` to handle the case where you have multiple azure-sdk-for-python directories? Or maybe I'm the only one who ever do 

```
mkdir azure-cli-for-python && cd azure-cli-for-python && git clone https://github.com/azure/azure-sdk-for-python"
```

:)